src/cryptography/hazmat/backends/openssl/backend.py

```diff
@@ -1037,6 +1037,18 @@ def load_elliptic_curve_public_numbers(self, numbers):
 
         return _EllipticCurvePublicKey(self, ec_cdata)
 
+    def elliptic_curve_exchange_algorithm_supported(
```

Comment:
Does this need to be separate from the regular `exchange_algorithm_supported` function?